all/pom.xml

```diff
@@ -213,5 +214,108 @@
                 </plugins>
             </build>
         </profile>
+
+        <profile>
+            <id>cloud</id>
+            <activation>
+                <activeByDefault>false</activeByDefault>
+            </activation>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.jackrabbit</groupId>
+                        <artifactId>filevault-package-maven-plugin</artifactId>
+                        <inherited>false</inherited>
+                        <executions>
+                            <execution>
+                                <id>generate-metadata-cloud</id>
+                                <goals>
+                                    <goal>generate-metadata</goal>
+                                </goals>
+                                <configuration>
+                                    <classifier>cloud</classifier>
+                                    <name>acs-aem-commons-all-cloud</name>
+                                    <packageType>container</packageType>
```

Comment:
This is redundant, I would only set configuration which differs from the default set outside the profile.